libbeat/common/cli/cli.go

```diff
@@ -7,11 +7,19 @@ import (
 	"github.com/spf13/cobra"
 )
 
+func rescue() {
+	if r := recover(); r != nil {
+		fmt.Fprintf(os.Stderr, "panic: %s\n", r)
```

Comment:
try to capture stack trace with `debug.Stack()`. I hope this really prints the complete trace, including the panic.